src/main/kotlin/sliv/tool/scene/view/FrameView.kt

```diff
@@ -31,25 +37,51 @@ class FrameView(
     }
 
     fun setFrame(frame: VisualizationFrame) {
-        //TODO: warn if image size doesn't equal to the frame size
+        currentJob?.cancel()
```

Comment:
will it be cancelled if reimport files to scene?

Reply:
I decided to create coroutine scope in the scene view and cancel it if data was reset